StringR.Store/Model/Racket.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  Racket.swift
+//  StringR.Store
+//
+//  Created by Marcus Christiansen on 13/09/2019.
+//  Copyright © 2019 StringR. All rights reserved.
+//
+
+import Foundation
+
+struct Racket {
+
+    var racketId: String
+    var brand: String
+    var modelName: String
+    var weight: Int
+    var main: Int
+    var cross: Int
+
+    enum RacketType: String {
+        case TENNIS = "Tennis"
+        case BADMINTON = "Badminton"
+        case SQUASH = "Squash"
+    }
```

Comment:
Is it possible to have the enum in a file for it self? That way we can use if for other purposes than 'Racket', eg. when showing economy.

Reply:
You can use them every where just type: "Racket.RacketType" ;)